src/fides/api/service/connectors/saas/connector_registry_service.py

```diff
@@ -70,15 +70,15 @@ def _load_connector_templates(self) -> None:
         logger.info("Loading connectors templates from the data/saas directory")
         for file in os.listdir("data/saas/config"):
             if file.endswith(".yml"):
-                config_file = os.path.join("data/saas/config", file)
-                config_dict = load_config(config_file)
-                connector_type = config_dict["type"]
-                human_readable = config_dict["name"]
-                user_guide = config_dict.get("user_guide")
-                authentication = config_dict["client_config"].get("authentication")
+                config_path = os.path.join("data/saas/config", file)
+                config = SaaSConfig(**load_config(config_path))
+
+                connector_type = config.type
+
+                authentication = config.client_config.authentication
```

Comment:
Refactoring to use a `SaaSConfig` instead of a `Dict` this way it's more consistent with the way it's done in other places in this file.

Reply:
that seems good 👍